src/mbgl/gfx/vertex_attribute.cpp

```diff
@@ -92,6 +92,13 @@ void VertexAttributeArray::resolve(const VertexAttributeArray& overrides, Resolv
     for (auto& kv : attrs) {
         delegate(kv.first, *kv.second, overrides.get(kv.first));
     }
+    // #if !defined(NDEBUG)
+    //     // Every override should match a defined attribute.
+    //     for (const auto& kv : overrides.attrs) {
+    //         const auto hit = attrs.find(kv.first);
+    //         assert(hit != attrs.end());
+    //     }
+    // #endif
```

Comment:
do we want to re-enable this at some point?

Reply:
Yeah, IIRC it fails on the missing pattern stuff.  It should really be `#if`d rather than commented, actually.